src/upcoming-components/FileUploadField/FileUploadField.tsx

```diff
@@ -237,18 +160,10 @@ export const FileUploadField = ({
     if (!fileObjects) return;
     const fileArray: Array<File> = Array.from(fileObjects);
 
-    /*
-     * 1. Copy existing files from state to a new variable
-     * 2. Append new files to this new array
-     * 3. If there is a maxFiles limit and if the more files have been selected than the max,
-     *    remove those over the limit from the array.
-     * 4. Set new state with existing and newly selected files
-     */
-
-    /* 1 */
```

Comment:
This is the kind of comment block the ticket was asking to place next to what they're referencing 👍